tests/migrations/test_graph.py

```diff
@@ -255,26 +240,30 @@ def test_validate_consistency(self):
         """
         Tests for missing nodes, using `validate_consistency()` to raise the error.
         """
-        # Build graph
+        # Case 1: missing parent node
```

Comment:
I think, given the way how you rewrite the test function, it's time to split the individual cases into individual test functions.

Reply:
Done